assets/css/src/components-public/_resets.scss

```diff
@@ -0,0 +1,82 @@
+// basic default styling
+// to allow for consistent styling between mailpoet plugin and user's theme
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  color: rgb(40, 48, 61);
```

Comment:
I'm just curious, what is the base for these styles, WP default admin theme?

Reply:
@wxa  Yes, the base style was gotten from the WP default admin theme